packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -72,8 +73,13 @@ module.exports = function(proxy, allowedHost) {
     quiet: true,
     // Reportedly, this avoids CPU overload on some systems.
     // https://github.com/facebookincubator/create-react-app/issues/293
+    // src/node_modules is not ignored to support absolute imports
+    // https://github.com/facebookincubator/create-react-app/issues/1065
     watchOptions: {
-      ignored: /node_modules/,
+      ignored: new RegExp(
+        `^(?!${path.normalize(paths.appSrc + '/')}).+[\\/]node_modules[\\/]`,
```

Comment:
Note that technically this stays in ejected projects forever. It looks extremely cryptic.
Can we extract this somewhere to `react-dev-utils`?